sim/global-types.ts

```diff
@@ -85,6 +85,8 @@ interface EventInfo {
 	from?: string;
 	/** Japan-only events can't be transferred to international games in Gen 1 */
 	japan?: boolean;
+	/** For Gen 3 event eggs to allow Pomeg glitched moves */
```

Comment:
I'd clarify it's specifically for Emerald (at least that's the assumption we have right now).

Reply:
Unless another obscure mechanic regarding event egg moves is found, I think it's unlikely that it will see more general use.